src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@
 #spring.datasource.username=postgres
 #spring.datasource.password=postgres
 #spring.jpa.generate-ddl=true
-server.port=8081
+server.port=8080
```

Comment:
I think this modification shouldn't be here

Reply:
You're right in the sense that the value stored in the repository shouldn't be changed. This update is just to fix a change that was inadvertently made in a previous commit. This was probably due to someone (perhaps myself :-( testing several services in a local machine, in this case each service should run in a different port. However, in a real deployment the services will run in separate containers. Thus, we defined that all Fogbow services should listen on port 8080. The deployment procedure does the appropriate port mapping between the virtual machine port and the container that runs the service.